src/Canvas.cc

```diff
@@ -248,6 +257,16 @@ NAN_METHOD(Canvas::ToBuffer) {
     return;
   }
 
+  if (info.Length() == 1 && info[0]->StrictEquals(Nan::New<String>("raw").ToLocalChecked())) {
```

Comment:
How about `info.Length() >= 1` to support `.toBuffer('raw', undefined, undefined)`? (might be used if someone wraps `.toBuffer`)